context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for people who want to create digital art.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursively creating shapes is easy because shapes are created inside rules.
 
 ## What is hard to do in this language? Why is it hard?
-
+It was very hard for me to learn it, taking well over an hour.  It requires
+memorization of syntax and avaiable rules unlike a simple program such as paint
+where you can see what is available for use.  Additional memorization is
+required to know what transforamtions you have available for your shape (such
+as using hue to color things is less obvious than something like color). Drawing
```

Comment:
That's a good point, but HSV is often a bit easier to reason about algorithmically and it better accounts for how we perceive color (i.e. green is much brighter to us).  While it's less intuitive for people not used to it, and probably hard to program with, I think it's understandable as a design choice that they would make HSV a first-class citizen.

I definitely think that RGB should also be first-class, however. Given it's prevalence and relative simplicity there isn't a good reason that it isn't better supported.